apps/resources/urls.py

```diff
@@ -4,11 +4,17 @@
 from pages.views import PageDetailView
 
 from .views import (
-    ResourceCategoryDetailView, ResourceCreateView, ResourceDetailView, ResourceUpdateView
+    ResourceCategoryDetailView, ResourceCreateView, ResourceDetailView, ResourceUpdateView,
+    admin_categorise_resources_view
 )
 
 app_name = 'resources'
 urlpatterns = [
+    url(
+        r'categorise-resources/$',
+        admin_categorise_resources_view,
+        name='admin-categorise-resources'
```

Comment:
`categorise` is freaking me out not having a `z` after working with `serializers` all week 😄